src/sentry/models/organization.py

```diff
@@ -205,6 +206,7 @@ def save(self, *args, **kwargs):
             self.save_with_snowflake_id(
```

Comment:
@maxiuyuan can there be a metric just before `self.save_with_snowflake_id()`? So we can compare the numbers pre and post creation of the snowflake ids. Ideally these numbers should match up. But it could be worth doing in case `self.save_with_snowflake_id()` borks.

Reply:
i have ` metrics.incr("snowflake_id_creation_failed", sample_rate=1)` in `save_with_snowflake_id()` fails. That should tell us if it fails without having to compare values

Reply:
although i wouldn't be opposed to adding a metric before for completeness

Reply:
Instead of a metric for the failure, should we be capturing an exception with sentry_sdk? We'll get more context that way.